src/components/__tests__/ars-test.jsx

```diff
@@ -13,11 +13,16 @@ describe("Ars", function() {
 
   describe("when the component renders", function() {
     let onChange  = sinon.spy()
-    let component = Test.renderIntoDocument(makeComponent({ onChange }))
+    let picked = 9
+    let component = Test.renderIntoDocument(makeComponent({ onChange, picked }))
 
     it ("has a selection component", function() {
       component.refs.should.have.property("selection")
     })
+
+    it ("migrates a single `picked` value to an array", function() {
+      component.state.should.have.property("picked").that.deep.equals([9])
```

Comment:
Also... woah I had no idea about `that`, or `deep`. I've always done:

`component.state.should.have.property("picked").should.eql([9])`

Yours is much cleaner!